.github/workflows/build-and-deploy.yml

```diff
@@ -364,8 +364,10 @@ jobs:
           file_text=$(cat "$FILE")
           if [ ! -z "$file_text" ]
           then
-            file_text="${file_text//$'\n'/%0A}"
-            echo "::set-output name=base_coverage_data::$file_text"
+            echo raw: "${file_text}"
+            # file_text="${file_text//$'\n'/%0A}"
+            # echo replaced: "${file_text}"
+            echo base_coverage_data="${file_text}" >> $GITHUB_OUTPUT
```

Comment:
@smneal It looks like we write to `$GITHUB_OUTPUT` but we never read it again (eg. reading the flakey spec data to make PR comments here https://github.com/DFE-Digital/apply-for-teacher-training/blob/2bd3d702a747d861b51db45409cd6fa6c255837b/.github/workflows/build-and-deploy.yml#L296). 
I think this will impact on reporting coverage data (probably not very important as it's rearely used) and also flakey spec reporting. @carlosmartinez are we using either of these PR reporting features any more?

Reply:
Also for multiline strings we may use: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings